x-pack/plugins/alerts/server/plugin.ts

```diff
@@ -161,6 +161,7 @@ export class AlertingPlugin {
   private eventLogService?: IEventLogService;
   private eventLogger?: IEventLogger;
   private readonly kibanaIndexConfig: Observable<{ kibana: { index: string } }>;
+  private kibanaBaseUrl: string | undefined;
```

Comment:
new instance variable so we can capture the path in `setup` and use it in `start`